Source/DafnyCore/Compilers/Rust/Dafny-compiler-rust-proofs.dfy

```diff
@@ -0,0 +1,362 @@
+
+
+
+
+
+include "../Dafny/AST.dfy"
+
+module {:extern "DCOMPProofs"} {:compile false} DCOMPProofs refines DCOMP {
+
+  ghost predicate IsDafnyIdTail(s: string) {
```

Comment:
nit: declaring this after `IsDafnyId` would make it much clearer what "tail" means

Reply:
I exchanged the two predicates in the next PR